sql-db/hibernate-fulltext-search/src/test/java/io/quarkus/ts/hibernate/search/MysqlMultitenantHibernateSearchIT.java

```diff
@@ -43,11 +44,6 @@ public class MysqlMultitenantHibernateSearchIT extends AbstractMultitenantHibern
             .withProperty("quarkus.datasource.company2.username", company2.getUser())
             .withProperty("quarkus.datasource.company2.password", company2.getPassword())
             .withProperty("quarkus.datasource.company2.jdbc.url", company2::getJdbcUrl)
-            .withProperty("quarkus.hibernate-search-orm.elasticsearch.hosts",
-                    () -> getElasticSearchConnectionChain(elastic.getURI(Protocol.HTTP)));
+            .withProperty("quarkus.hibernate-search-orm.elasticsearch.hosts", () -> elastic.getURI().toString());
```

Comment:
It seems, that this is not needed
```suggestion
            .withProperty("quarkus.hibernate-search-orm.elasticsearch.hosts", () -> elastic.getURI());
```

Reply:
> It seems, that this is not needed

if I applied this suggestion, it would result in compilation error, the method signature is `    public T withProperty(String key, Supplier<String> value)` and there is no override for URILike.

Reply:
My bad, I didn't notice that you use `getElasticSearchConnectionChain` for such URIs. Any reason why you do it in some places, but not in others?